reporting-procedures.md

```diff
@@ -10,22 +10,16 @@ If you find that you need to make a report, and you cannot find the appropriate
 
 *General PSF reporting procedure:*
 
-The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [LINK](). 
+The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are). 
```

Comment:
"this alias is listed at" -> "this alias are listed at"

Reply:
```suggestion
The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias are listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are). 
```